include/envoy/grpc/async_client.h

```diff
@@ -91,6 +104,20 @@ class AsyncRequestCallbacks {
                          Tracing::Span& span) PURE;
 };
 
+class AsyncRequestCallbacks : public RawAsyncRequestCallbacks {
```

Comment:
Do we really need this class? Can `RawAsyncRequestCallbacks` act as this?

Reply:
This class provides support for onSuccessUntyped which in turn provides support for onSuccess (the typed version).

Reply:
Done